WolvenKit/Layout/inkWidgets/inkBaseImageControl.cs

```diff
@@ -177,7 +180,9 @@ protected virtual void DrawImage(SizeF size)
                 {
                     attributes.SetWrapMode(System.Drawing.Drawing2D.WrapMode.TileFlipXY);
 
-                    if (Rect.Top != 0)
+                    DrawNineSlicedImage(gfx, destBitmap, sourceBitmap, attributes);
+
+                    /*if (Rect.Top != 0)
```

Comment:
Can this code go?

Reply:
Yeah, just kept it for reference if I fucked up somehow